Makefile

```diff
@@ -139,9 +144,9 @@ endif
 endif
 
 ifneq ($(filter release,$(TAGS)),)
-DEPS += -lwren
+#DEPS += -lwren
 else ifneq ($(filter debug,$(TAGS)),)
-DEPS += -lwrend
+#DEPS += -lwrend
```

Comment:
Fixed in 4847c6c

Reply:
We need to add a `-DDEBUG` flag to the Wren params if we are building in debug mode, so that we can switch between the release and debug modes of the Wren VM. (This is only for engine development really, but still important to me :P

Reply:
Fixed in 7d03937 & 179456c